plugins/SegmentEditor/stylesheets/segmentation.less

```diff
@@ -125,11 +125,18 @@ div.scrollable {
     color: #505050;
     text-align: right;
     padding: 3px 7px 0 0;
-
+    display: grid;
+    grid-template-columns: 28% 28% 44%;
```

Comment:
Keep in mind that this is not supported in Internet explorer 11 which is (unfortunatly) still supported:
https://caniuse.com/css-grid

That said, support will probably droped in 4.6.0 anyway (because of #17940), so this isn't that important

Reply:
@peterhashair in Matomo 4.5 we ideally still support IE 11. Is this change needed?